### PR TITLE
feat: support record screen view event manually

### PR DESCRIPTION
**Pull request by zhu-xiaowei**
- [x] PR title and description conform to [Pull Request](https://github.com/awslabs/clickstream-android/blob/main/CONTRIBUTING.md#pull-request-guidelines) guidelines.

*Description of changes:*
1. support record screen view event manually

*How did you test these changes?*
(Please add a line here how the changes were tested)
added new unit test case for verify the manually screen view events record

*Documentation update required?*
- [ ] No
- [x] Yes (Please include a PR link for the documentation update)

*General Checklist*
- [x] Added Unit Tests

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +204,27 @@ ClickstreamEvent event = ClickstreamEvent.builder()
 ClickstreamAnalytics.recordEvent(event);
 ```
 
+#### Record Screen View events manually
+
+By default, SDK will automatically track the preset `_screen_view` event when Activity triggers `onResume`.
+
+You can also manually record screen view event whether or not automatic screen view tracking is enabled, add the following code to record a Screen View event with two attributes.
```
Comment (zxkane) [inline README.md:211]: screen view event -> screen view events

File: README.md [hunk 2]
```diff
@@ -204,6 +204,27 @@ ClickstreamEvent event = ClickstreamEvent.builder()
...
+
+* `SCREEN_NAME` Required. Your screen's name.
+* `SCREEN_UNIQUE_ID` Optional. set the hashcode of your Activity, Fragment or View. And if you do not set, SDK will sets a default value based on the current Activity's hashcode.
```
Comment (zxkane) [inline README.md:214]: set the hashcode of your Activity, Fragment, or View. If you do not set, SDK will set a default value based on the current Activity's hashcode.

**Review by zxkane — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zhu-xiaowei) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (zhu-xiaowei) [reply README.md]: fixed

**Pull request merged**